### PR TITLE
Feature/test with postgres

### PR DESCRIPTION
**Pull request by drnlm**
We shouldn't test against only 1 database, especially as we have generally deployed with postgres rather than sqlite.

I still need to investigate why Django 1.7 doesn't like the kv migration.

### CODE REVIEW

**Comment by stefanor** [comment]
:+1: to the plan. But yes, we need to make it pass.

File: wafer/kv/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0006_require_contenttypes_0002'),
```
Comment (hodgestar) [inline wafer/kv/migrations/0001_initial.py]: Maybe write this as `django.contrib.auth`?

Comment (hodgestar) [inline wafer/kv/migrations/0001_initial.py]: Or maybe `migrations.swappable_dependency(settings.AUTH_USER_MODEL)`?

**Comment by hodgestar** [comment]
We do now run a lot of different builds -- too many yet?

**Comment by drnlm** [comment]
It's debatable how much benefit there is to still testing with python 3.3, so we could drop those builds.

Since Django 1.7 has reached end of life, there is also an argument that we shouldn't bother with it anymore.

File: wafer/kv/migrations/0001_initial.py (hunk 1 above)
Comment (drnlm) [inline wafer/kv/migrations/0001_initial.py]: Thanks for pointing out migrations.swappable_dependency - that saved me a lot of time digging through the migration documentation.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**